data/mods/Magiclysm/Spells/magus.json

```diff
@@ -63,5 +63,107 @@
     "base_casting_time": 600,
     "energy_source": "MANA",
     "base_energy_cost": 350
+  },
+  {
+    "id": "magus_eldritch_blast",
+    "type": "SPELL",
+    "name": "Eldritch Blast",
+    "description": "A blast of concentrated magical power that obliterates a large area.",
+    "valid_targets": [ "hostile", "ground" ],
+    "effect": "projectile_attack",
+    "spell_class": "MAGUS",
+    "energy_source": "MANA",
+    "base_casting_time": 150,
+    "base_energy_cost": 200,
+    "difficulty": 5,
+    "max_level": 20,
+    "min_damage": 20,
+    "max_damage": 80,
+    "damage_increment": 3,
+    "damage_type": "none",
+    "min_aoe": 1,
+    "max_aoe": 5,
+    "aoe_increment": 0.5,
+    "min_range": 5,
+    "max_range": 20,
+    "range_increment": 1
+  },
+  {
+    "id": "magus_mana_bolt",
+    "type": "SPELL",
+    "name": "Mana Bolt",
+    "description": "A bolt of magical power that only damages your foes.",
+    "valid_targets": [ "hostile" ],
+    "effect": "projectile_attack",
+    "spell_class": "MAGUS",
+    "energy_source": "MANA",
+    "base_casting_time": 100,
+    "base_energy_cost": 150,
+    "difficulty": 3,
```

Comment:
this should be much higher. 3 means this spell is basically strictly better than magic missile in every way.

Reply:
Except that it's a projectile and won't pass through obstacles, it also costs 3x as much. This is another effort to compete with basic firearms like a shotgun that can easily do 50 points or more.

Reply:
i'd still bump it up to 5 ish

Reply:
Let's settle on 3 or 4, because if I make it 5, it's going to look useless compared to mana blast which does 80 damage to 3 aoe at dif 5 and nobody will use it.

Reply:
actually, you bring up a good point; this spell probably shouldn't cap out at a much lower maximum damage than the aoe version of it. i say bump it up to 5, but bring its max damage up to 120 or so

Reply:
Agreed, I wanted to bridge the gap between higher level beam/blast spells and MM.